app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -86,38 +86,38 @@ The main visa route for non-UK citizens wishing to teach in England is the skill
 
 You’ll be able to apply for your skilled worker visa if:
 
-* you have a job offer from a school that is a Home Office licensed visa sponsor 
+* you have a job offer from a school that is a Home Office licensed visa sponsor
 * you have a certificate of sponsorship from this school, confirming they have offered you a job and will sponsor you
 * [you meet certain standards in speaking, reading, writing and understanding English](https://www.gov.uk/skilled-worker-visa/knowledge-of-english)
 * your salary meets the requirement for a skilled worker visa
 
 [Learn more about applying for a teaching job in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
 
 ### Salary requirement for a skilled worker visa
- 
+
 The salary you must be paid to get a visa varies according to  whether you are employed as a ‘qualified’ or ‘unqualified’ teacher. This depends on your level of experience and qualifications – your employing school can explain whether the role you are applying for is qualified or unqualified. The salary requirement also varies depending on the region you are applying to work in.
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £36,745
-* outer London, you must earn £34,514
-* London fringe, you must earn £31,350
-* the rest of England, you must earn £30,000
+* inner London, you must earn $salaries_skilledworkervisateacherminfulltime_innerlondon$
+* outer London, you must earn $salaries_skilledworkervisateacherminfulltime_outerlondon$
+* London fringe, you must earn $salaries_skilledworkervisateacherminfulltime_londonfringe$
+* the rest of England, you must earn $salaries_skilledworkervisateacherminfulltime_restofengland$
```

Comment:
For consistency, I suggest we have the values like below so the location and min and max are at the end as that's what's in the values list eg salaries_example_min.
 
salaries_skilledworkervisateacher_fulltime_innerlondonmin

Also, do we need full-time in every variable if there aren't many part-time options? Can we just assume all courses are full-time and then use parttime in the variable for the part-time ones? 

What do you reckon @gemmadallmandfe? Also do you know if Myles had a list of all the salary variables? It's not on the values list, guessing that's because there are so many of them!

Reply:
Agree that we could make these more consistent in format - but I think we should shorten as well - let's discuss

Reply:
Think it makes sense to put min on the end for consistency and also removing visa is fine

However, I think ft and pt are actually harder to understand now? But I don't feel too strongly about this

Reply:
NB it is a bit inconsistent - 

* `skilledworkertraineeptmin_restofengland` £23,200

* `skilledworkerteacherptmin` £23,200

I think we should have `_restofengland` on both or neither?

Reply:
went with keeping _restofengland in for consistency with the london terms